api/hooks/driver.js

```diff
@@ -39,8 +39,8 @@ function Driver(sails) {
      * @param {Function} callback The initialize callback
      */
     initialize(callback) {
-      sails.after('hook:config:loaded', () => {
-        MachineService.initialize()
+      sails.after('hook:orm:loaded', () => {
+        return MachineService.initialize()
```

Comment:
We need to wait for the `hook:config:loaded` event. `MachineService` uses the `ConfigService`.